.buildkite/pipeline.yml

```diff
@@ -897,8 +897,8 @@ steps:
       slurm_ntasks: 1
       slurm_gres: "gpu:1"
 
-  - label: "gpu_risingbubble"
-    key: "gpu_risingbubble"
+  - label: "gpu_risingbubble_tutorial"
+    key: "gpu_risingbubble_tutorial"
     command:
       - "mpiexec julia --color=yes --project tutorials/Atmos/risingbubble.jl "
```

Comment:
Do we still need to run the tutorial version for CI?

Reply:
I'm fine with removing it. @akshaysridhar ?

Reply:
I removed this from CI.